README.md

```diff
@@ -30,19 +31,29 @@ in the bundle (perhaps because of out-of-date dependencies).
     ```
     source-map-explorer --json foo.min.js
     {
-      "node_modules/browserify/node_modules/browser-pack/_prelude.js": 463,
-      "bar.js": 62,
-      "foo.js": 137
+      "results": [
+        {
+          "bundleName": "tests\\data\\foo.min.js",
+          "files": {
+            "node_modules/browserify/node_modules/browser-pack/_prelude.js": 463,
+            "dist/bar.js": 97,
+            "dist/foo.js": 137,
+            "<unmapped>": 0
+          }
+        }
+      ]
     }
     ```
 
 * `--tsv`: output tab-delimited values instead of displaying a visualization:
 
     ```
     source-map-explorer --tsv foo.min.js
-    Source	Size
-    dist/bar.js	62
-    dist/foo.js	137
+    Source  Size
```

Comment:
Looks like the values and column headings got out of sync here.

Reply:
Hm, it looks fine in view mode
![image](https://user-images.githubusercontent.com/4221449/57008325-5a29cf00-6c00-11e9-8dca-a93362646a97.png)

Reply:
The issue is in your screenshot, too: Under "Source" you're listing sizes, but under "Size" you're listing sources :)

Reply:
Other than this, my feedback was all style issues. No need to block release on them if you want to go forward. Great work on this!

Reply:
I see now, I will fix it

Reply:
Fixed